command/catalog_command.go

```diff
@@ -0,0 +1,50 @@
+package command
+
+import (
+	"strings"
+
+	"github.com/mitchellh/cli"
+)
+
+var _ cli.Command = (*CatalogCommand)(nil)
+
+type CatalogCommand struct {
+	BaseCommand
+}
+
+func (c *CatalogCommand) Run(args []string) int {
+	return cli.RunResultHelp
+}
+
+func (c *CatalogCommand) Help() string {
+	helpText := `
+Usage: consul catalog <subcommand> [options] [args]
+
+  This command has subcommands for interacting with Consul's catalog. The
+  catalog should not be confused with the agent, although the APIs and
+  responses may be similar.
+
+  Here are some simple examples, and more detailed examples are available
+  in the subcommands or the documentation.
+
+  List all datacenters:
+
+      $ consul catalog datacenters
```

Comment:
Yeah I kind of like the verbs (like https://www.consul.io/docs/commands/operator/area.html), we could start out that way so we get help with the top-level command showing help and we've got a spot for more things later like `deregister`:

```
consul catalog datacenters list
consul catalog nodes list
consul catalog services list
```

If we start out with `consul catalog datacenters` running list, there's no where to park the help for the `datacenters` subcommands later, for example.